src/js/containers/award/table/FinancialSystemTableContainer.jsx

```diff
@@ -88,7 +88,8 @@ export class FinancialSystemTableContainer extends React.Component {
                     value: awardId
                 }
             ],
-            order: [`${sortDirection}${this.state.sort.field}`],
+            order: this.state.sort.field === 'submission__reporting_fiscal_year' ?
+                [`${sortDirection}${this.state.sort.field}`, `-submission__reporting_fiscal_quarter`] : [`${sortDirection}${this.state.sort.field}`],
```

Comment:
Use `sortDirection` instead of hard-coding `-` so that the quarters' ascending/descending direction will match the years'